prometheus_client/openmetrics/exposition.py

```diff
@@ -67,24 +67,87 @@ def generate_latest(registry):
-                    exemplarstr = ''
+                    exemplarstr = ''   
+                        
                 timestamp = ''
                 if s.timestamp is not None:
                     timestamp = f' {s.timestamp}'
+                
+                native_histogram = ''
+                positive_spans = ''
+                positive_deltas = ''
+                negative_spans = ''
+                negative_deltas = ''
+                pos = False
+                neg = False
+                     
+                if s.native_histogram:                
+                    # Initialize basic nh template
+                    nh_sample_template = '{{count:{},sum:{},schema:{},zero_threshold:{},zero_count:{}'
+
+                    args = [
+                        s.native_histogram.count_value,
+                        s.native_histogram.sum_value,
+                        s.native_histogram.schema,
+                        s.native_histogram.zero_threshold,
+                        s.native_histogram.zero_count,
+                    ]
+
+                    # Signal presence for pos/neg spans/deltas
+                    pos = False
+                    neg = False
+
+                    # If there are pos spans, append them to the template and args
+                    if s.native_histogram.pos_spans:
+                        positive_spans = ','.join([f'{ps[0]}:{ps[1]}' for ps in s.native_histogram.pos_spans])
+                        positive_deltas = ','.join(f'{pd}' for pd in s.native_histogram.pos_deltas)
+                        nh_sample_template += ',positive_spans:[{}]'
+                        args.append(positive_spans)
+                        pos = True
+
+                    # If there are neg spans exist, append them to the template and args
+                    if s.native_histogram.neg_spans:
+                        negative_spans = ','.join([f'{ns[0]}:{ns[1]}' for ns in s.native_histogram.neg_spans])
+                        negative_deltas = ','.join(str(nd) for nd in s.native_histogram.neg_deltas)
+                        nh_sample_template += ',negative_spans:[{}]'
+                        args.append(negative_spans)
+                        neg = True
+
+                    # Append pos deltas if pos spans were added
+                    if pos:
+                        nh_sample_template += ',positive_deltas:[{}]'
+                        args.append(positive_deltas)
+
+                    # Append neg deltas if neg spans were added
+                    if neg:
+                        nh_sample_template += ',negative_deltas:[{}]'
+                        args.append(negative_deltas)
+
+                    # Add closing brace
+                    nh_sample_template += '}}'
+
+                    # Format the template with the args
+                    native_histogram = nh_sample_template.format(*args)    
+
+                value = ''    
+                if s.value is not None or not s.native_histogram:
```

Comment:
it should be enough to write
```suggestion
                if not s.native_histogram:
```

At least that's what we do in Prometheus. Type checks should prevent having None in the value.